test/functional/mempool_packages.py

```diff
@@ -100,6 +100,11 @@ def run_test(self):
             entry = self.nodes[0].getmempoolentry(x)
             assert_equal(entry, mempool[x])
 
+            # Check that getmempoolspendingtx is consistent with getrawmempool
```

Comment:
It looks like there aren't any dedicated test files for `getmempoolentry`, `getmempoolancestors` and `getmempooldescendants` (I wanted to insert my tests in such a file).

Should I create a new `rpc_getmempoolpendingtx.py` to put more tests (e.g. some failure cases)?

Reply:
Yeah, that's probably the move. "rpc_mempool_info.py" would be a good place to put tests for all of these RPCs.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/24408/commits/4ef5a16c306bb4784fb645d1df51fe0efeadcb06, I have only added tests for my new RPC for now, future PRs can add tests for `getmempoolentry`, `getmempoolancestors` and `getmempooldescendants` here if that makes sense.